main.go

```diff
@@ -146,40 +145,41 @@ func handle(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		} else if errors.Is(err, fs.ErrNotExist) {
-			// go build
-			args := []string{"build", "-o", filepath.Join(output, "main.wasm")}
+			// go run
+			args := []string{"run", "-exec", "cp"}
 			if *flagTags != "" {
 				args = append(args, "-tags", *flagTags)
 			}
 			if *flagOverlay != "" {
 				args = append(args, "-overlay", *flagOverlay)
 			}
-			if len(flag.Args()) > 0 {
+			if flag.NArg() > 0 {
```

Comment:
We can give an error when `flag.NArgs() > 1`

Reply:
Was this reoslved?

Reply:
On second though, `go run` doesn't warn anything when giving multiple packages:

```
go run -tags=example github.com/hajimehoshi/ebiten/v2/examples/blocks@latest github.com/hajimehoshi/ebiten/v2/examples/2048@latest
```

The way in which dependencies are resolved might slightly be different, but just ignoring the second and following arguments seems fine so far.